src/screenshot.js

```diff
@@ -45,26 +45,25 @@ function getScreenshotFunc() {
         const screenshotFile = getScreenshotFileName();
         if (res.constructor.name == "Promise") {
           res.then((data) => {
-            errorHandling( () => {
-              fs.writeFileSync(screenshotFile, data);
-              resolve(path.basename(screenshotFile));
-            });
+            writeScreenshotWithErrorHandling(data, screenshotFile, resolve);
           });
         } else {
-          errorHandling( () => {
-            fs.writeFileSync(screenshotFile, res);
-            resolve(path.basename(screenshotFile));
-          });
+          writeScreenshotWithErrorHandling(res, screenshotFile, resolve);
         }
       });
     };
   }
   return defaultScreenshotWriter;
 }
 
-function errorHandling(action) {
+function writeScreenshotWithErrorHandling(data, screenshotFile, resolver) {
```

Comment:
`writeScreenshotWithErrorHandling` suggests that there is some error handling, but I don't see any error handling. can we rename this?